databricks/koalas/series.py

```diff
@@ -1748,6 +1748,71 @@ def reindex(self, index: Optional[Any] = None, fill_value: Optional[Any] = None,
             self.name
         )
 
+    def reindex_like(self: "Series", other: Union["Series", "DataFrame"]) -> "Series":
+        """
+        Return a Series with matching indices as other object.
+
+        Conform the object to the same index on all axes. Places NA/NaN in locations
+         having no value in the previous index.
```

Comment:
We should remove an extra space before `having`; otherwise, the doc format breaks:

<img width="233" alt="Screen Shot 2020-11-04 at 10 43 14 AM" src="https://user-images.githubusercontent.com/506656/98155141-a35c7e00-1e8a-11eb-8195-1bbc02a82bdc.png">

Reply:
Oh yes thanks !